ddp-workspace/projects/ddp-dsm-ui/src/app/field-datepicker/field-datepicker.component.ts

```diff
@@ -58,7 +59,7 @@ export class FieldDatepickerComponent implements OnInit, OnChanges {
   }
 
   private handleInput(dateString: string): void {
-    if (dateString !== 'N/A' && dateString !== 'Not Found') {
+    if (dateString !== 'N/A' && dateString !== 'Not Found' && dateString !== this.defaultDate) {
```

Comment:
The date values 'N/A' , 'Not Found', '1000-01-01' are used in several files.
It would be nice to have an enum for values.

Reply:
`(dateString !== 'N/A' && dateString !== 'Not Found' && dateString !== this.defaultDate)`
can be shortened like
`!['N/A', 'Not Found', this.defaultDate].includes(dateString)`